syntax="proto3"; //消息类型命名规范：使⽤驼峰命名法，⾸字⺟⼤写。
package mynumber;


//定义联系人消息
message MyNumber{
    optional int32  opt_number=1;  // optional
             int32  imp_number=2;  // implicit
}
